pkg/primitive/log.go

```diff
@@ -0,0 +1,149 @@
+package primitive
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hyperifyio/gnd/pkg/log"
+)
+
+// Log levels
+const (
+	LogError = iota
+	LogWarn
+	LogInfo
+	LogDebug
+)
+
+// Log represents the log primitive
+type Log struct{}
+
+// Name returns the name of the primitive
+func (l *Log) Name() string {
+	return "/gnd/log"
+}
+
+// Execute runs the log primitive
+func (l *Log) Execute(args []interface{}) (interface{}, error) {
+	// Case 1: No arguments - use _ as an array
+	if len(args) == 0 {
+		log.Printf(log.Info, "_")
+		return nil, nil
+	}
+
+	// Case 2: Single argument - could be a string or an array from _
+	if len(args) == 1 {
```

Comment:
If we only have one argument, that means the user requested to write empty log level message. We should probably make an error.